.github/workflows/tox.yml

```diff
@@ -31,7 +31,7 @@ jobs:
     - name: Test with tox
       run: |
         pyenv="py$(echo "${{ matrix.python-version }}" | tr -d '.')"
-        tox -e ${pyenv}-test,${pyenv}-rapidjson,flake8,lint
+        tox -e ${pyenv}-test,${pyenv}-rapidjson,flake8,lint,black
```

Comment:
Appearently this also needs to be handled separately, as `black` seems to have dropped 3.5 support ...

Reply:
Yes unfortunately...